src/test/shared/codelens/localLambdaRunner.test.ts

```diff
@@ -12,11 +12,17 @@ import { DebugConfiguration } from '../../../lambda/local/debugConfiguration'
 import * as localLambdaRunner from '../../../shared/codelens/localLambdaRunner'
 import * as fsUtils from '../../../shared/filesystemUtilities'
 import { ChildProcessResult } from '../../../shared/utilities/childProcess'
+import { ExtensionDisposableFiles } from '../../../shared/utilities/disposableFiles'
+import { FakeExtensionContext } from '../../fakeExtensionContext'
 import { FakeChannelLogger } from '../fakeChannelLogger'
 import { assertRejects } from '../utilities/assertUtils'
 
 describe('localLambdaRunner', async () => {
     let tempDir: string
+    before(async () => {
+        await ExtensionDisposableFiles.initialize(new FakeExtensionContext())
```

Comment:
Does this conflict with the call in `src/shared/utilities/disposableFiles.ts` ? Since the tests are running in the same process and `ExtensionDisposableFiles.initialize` only allows one instance, it could cause an exception: https://github.com/aws/aws-toolkit-vscode/pull/1132#issuecomment-638999234

Reply:
Looks like an oversight in how the test was initially set up. 
The ExtensionDisposableFiles instance should be reset (to null) `after` this test suite. ExtensionDisposableFiles would need to have a method added to support this.